api/tests/routes/saml/educonnect_test.py

```diff
@@ -181,6 +181,8 @@ def test_connexion_date_missing(self, mock_get_educonnect_saml_client, client, c
         # set user_id in redis as if /saml/educonnect/login was called
         user = users_factories.UserFactory(email=self.email)
         app.redis_client.set(f"{self.request_id_key_prefix}{self.request_id}", user.id)
+        # We don't need the birth date to be exact, it just needs to be less than 18 years
+        birth_date = (datetime.date.today() - datetime.timedelta(days=17 * 365)).strftime("%Y-%m-%d")
```

Comment:
Suggestion pour la date ISO : 
```suggestion
        birth_date = (datetime.date.today() - datetime.timedelta(days=17 * 365)).isoformat()
```